website/docs/core-concepts/building-ui/designing-an-application/app-navigation.md

```diff
@@ -0,0 +1,87 @@
+# App Navigation
+
+We have introduced new navigation styles with better a default navigation - Top (Stacked), a variant for Top (Inline), and a collapsible Sidebar. You can now configure your app's navigation by opening the navigation settings tab inside the app settings pane and observe how your app with the selected navigation settings will look side by side as you change them.
+
+<figure>
+  <img src="/img/app-navigation-new-default.png" alt="The new default navigation - Top (Stacked)"/>
+  <figcaption align="center"><i>The new default navigation - Top (Stacked)</i></figcaption>
+</figure>
+
+## Navigation settings
+
+:::info
+Configuring the navigation is an application-level change.
+:::
+
+You can configure the navigation for your app using -
+
+- Open the `App settings`.
+- Click on `Navigation`.
+- Select the desired settings.
+- The settings will auto save.
+
+<figure>
+  <img src="/img/app-navigation-settings.gif" alt="Changing navigation settings"/>
+  <figcaption align="center"><i>Changing navigation settings</i></figcaption>
+</figure>
+
+### Configuration
+
+The options available of configuring the navigation settings are as follows -
+
+| Property         | Description                                                                                           | Default | Available Options                                                      |
+| ---------------- | ----------------------------------------------------------------------------------------------------- | ------- | ---------------------------------------------------------------------- |
+| Show navbar      | Toggles the visibility of the navbar. If turned off, the navbar won't be visible on the deployed app. | On      | -                                                                      |
+| Orientation      | Changes if the navbar should be on top or side.                                                       | Top     | Top and Side. [Read more here.](app-navigation.md#orientation)         |
+| Variant          | Only available when the orientation is set to Top. Available options are Stacked and Inline.          | Stacked | Stacked and Inline. [Read more here.](app-navigation.md#variant)       |
+| Background color | Changes the background color of the navigation.                                                       | Light   | Light and Theme. [Read more here.](app-navigation.md#background-color) |
+| Show sign in     | Toggles the visibility of the sign-in button when someone who is not logged in views the app.         | On      | -                                                                      |
```

Comment:
With these prop should we also add the scope, i.e. where the change is going to be visible

Reply:
Can you explain a bit more about what the scope means here? Can you please give an example? I didn't quite get you.

Reply:
Sure, my suggestion is to explicitly call out the scope for a function. 
ex: `Show sign in` is only going to take effect on the single app having this config. 
If it makes sense we can use this as an opportunity to show the advantage for business editions users

Reply:
Gotcha, all of these options have an application-wide scope, which we have earlier mentioned and I'm defining it a bit more clearly as you suggested in the [above comment](https://github.com/appsmithorg/appsmith-docs/pull/1082#discussion_r1155830162). Would that complete the purpose of sharing the scope?